apps/blade/src/app/admin/events/page.tsx

```diff
@@ -0,0 +1,37 @@
+import { redirect } from "next/navigation";
+
+import { auth } from "@forge/auth";
+
+import { isDiscordAdmin } from "~/app/utils";
+import { SIGN_IN_PATH } from "~/consts";
+import { HydrateClient } from "~/trpc/server";
+import { EventsTable } from "./_components/events-table";
+
+export default async function Events() {
+  // Check if the user is authenticated
+  const session = await auth();
+  if (!session) {
+    redirect(SIGN_IN_PATH);
+  }
+
+  // Check if the user has access to Blade
+  const isAdmin = await isDiscordAdmin(session.user);
+  if (!isAdmin) {
+    redirect("/dashboard");
```

Comment:
this is just so that I don't have to change it later so change it to "/", this is where the dash will sit if they are authenticated